README.md

```diff
@@ -4,59 +4,49 @@ Kute App
 
 **Kute** is an app that makes navigation and transportation easier and more efficient for people travelling to the same destination.People can easily navigate and identify public transport services around them without having to look through the schedules and timings.Travellers can share their location and tackle the problems of ever-changing schedules together.
 
-![enter image description here](http://res.cloudinary.com/raghavp/image/upload/v1513531690/kute1_bhhvhj.png)
-
-![enter image description here](http://res.cloudinary.com/raghavp/image/upload/v1513531691/kute2_vtvneu.png)
+<img src="http://res.cloudinary.com/raghavp/image/upload/v1513531690/kute1_bhhvhj.png" alt="screenshot 1" height="500"> <img src="http://res.cloudinary.com/raghavp/image/upload/v1513531691/kute2_vtvneu.png" alt="screenshot 2" height="500">
 
 ----------
-
-
-Installation
+Installation instructions
 -------------
 
-You can install and run **Kute App** easily on your android device or emulator using the following instructions
-
-1)Download the zip file from the repository at 
-https://github.com/scorelab/kute/tree/c_in_refine or,use the following git command to clone the file 
+- Download the zip file from the repository at 
+https://github.com/scorelab/kute/tree/c_in_refine oruse the following git command to clone the file 
 > git clone https://github.com/scorelab/kute.git
 
-2)Next set up Android Studio on your device,using the following link 
+- Next set up Android Studio on your device,using the following link 
 https://developer.android.com/studio/install.html
 
-3)Start Android studio and download the SDK's
+- Start Android studio and download the SDKs
 
-4)Set Up your emulator in Android Studio,you can do so by going to Tools>Android>AVD Manager,next you will be prompted with instructions to set up your emulator.
+- Set Up your emulator in Android Studio,you can do so by going to `Tools>Android>AVD Manager`,next you will be prompted with instructions to set up your emulator.
 
-5)Go to the zip file or navigate to the directory using the "Cd" command and launch the project
+- Go to the zip file or navigate to the directory using the `cd` command and launch the project
 
-6)The project will build automatically,there might be some errors but android studio will provide you with the problem and even give you an option to install the required components causing the error.
+- The project will build automatically,there might be some errors but android studio will provide you with the problem and even give you an option to install the required components causing the error.
```

Comment:
Can you make this as follows?
- The project should build automatically. If there were any errors, the android studio will provide you with problems and options to install the required components causing the error.